src/OrchardCore.Modules/OrchardCore.Email.Azure/Startup.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OrchardCore.Admin;
+using OrchardCore.Email.Azure.Controllers;
+using OrchardCore.Email.Azure.Services;
+using OrchardCore.Email.Services;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Modules;
+using OrchardCore.Mvc.Core.Utilities;
+using OrchardCore.Navigation;
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Email.Azure;
+
+public class Startup : StartupBase
+{
+    private readonly AdminOptions _adminOptions;
+    private readonly ILogger _logger;
+    private readonly IShellConfiguration _configuration;
+
+    public Startup(
+        IOptions<AdminOptions> adminOptions,
+        ILogger<Startup> logger,
+        IShellConfiguration configuration)
+    {
+        _adminOptions = adminOptions.Value;
+        _logger = logger;
+        _configuration = configuration;
+    }
+
+    public override int Order => 10;
```

Comment:
I don't we need to set the `Order`

Reply:
Yes you need to set the order here to ensure that the Azure service take presence over the Email when it is activated.  

I am saying this would have been avoided if you used the providers design I suggested originally.

Reply:
I don't think I need it now, it was for a reason, you can check the SMTP